src/shared/components/case-editor/domain/wizard-page-field-complex-mask.model.ts

```diff
@@ -0,0 +1,10 @@
+import { Orderable } from '../../../domain/order/orderable.model';
+
+export class ComplexFieldMask implements Orderable {
+  complex_field_id: string;
```

Comment:
this can be misleading imho. It's not the complex field id. This identifies one of the complex field elements, and it's a dot separated path rather than its id.
Shall we rename this?